admin/app/models/session.js

```diff
@@ -79,6 +79,11 @@ export default class Session extends Model {
     return _formatHumanReadableLocaleDateTime(this.resultsSentToPrescriberAt);
   }
 
+  @computed('resultsSentToPrescriberAt')
+  get resultsToSendToPrescriber() {
+    return this.isFinalized && !this.resultsSentToPrescriberAt;
+  }
```

Comment:
Le nom de ce computed n'est pas clair... Si ça retourne un booléen, il devrait commencer par un verbe style is... ou have... ou does..... A la lecture, on dirait que ce computed est supposé nous donner les résultats transmis au prescripteur

Reply:
`isResultsSentToPrescriber` ?

Reply:
Ben je sais pas on dirait plutôt que tu cherches le contraire non ? vu la condition avec le `!this.resultsSentToPrescriberAt`

Reply:
`wereResultsSentToPrescriber` pour la version positive. Effectivement c'est l'inverse de la condition actuelle, mais c'est plutôt une bonne idée de rester positif dans les fonctions qui renvoient un booléen.